data/clients.json

```diff
@@ -787,6 +787,13 @@
 			"url": "https://github.com/mk-fg/acme-cert-tool",
 			"category": "Python",
 			"acme_v2": "true"
+		},
+		{
+			"name": "ght-acme.sh",
+			"url": "https://github.com/bruncsak/ght-acme.sh",
+			"category": "Bash",
+ 			"comments": "(batch update of http-01 and dns-01 challenges is available)",
```

Comment:
I'm not sure that I understand this comment. I suspect all of the listed ACME clients can handle batching challenge responses for a multiple identifier order.

The other Bash category ACME clients only use the comments field for their shell compatibility notes and I suspect that would be a better choice here for `ght-acme` too if applicable. Otherwise I think leaving the comments empty is the best choice. What do you think @bruncsak?

Reply:
There are ACME clients which are not supporting batch update, see that issue: https://github.com/Neilpang/acme.sh/issues/1173 . I specifically worked on this feature to make it available in ght-acme.sh. Before that I was ready to drop my client and switch to the other one, but the lack of that feature on the other client pushed me to continue.

Concerning the shell compatibility, I do not know is it important. I left it out because I tested on a very old UNIX system's shell, and it deadly failed. The script needs a more decent than Bourne shell, probably at least Korn level.

Reply:
> There are ACME clients which are not supporting batch update, see that issue: Neilpang/acme.sh#1173 .

Ah, interesting. I see what you mean now but I wonder if most people will be able to understand from the comment alone. I don't feel strongly about it so I will resolve my feedback. Thanks